bakalari_homework_tasklist/templates/list_homeworks.html

```diff
@@ -160,7 +160,6 @@
         }
 
         body.big table.details-table {
-            float: left;
```

Comment:
Why to remove this? The table will become full-width, which is IMO unnecessary...

Reply:
Well at least on my computer (firefox) it keeps the table same but pushes the text down so parts of it are not next to the table but all is below

Reply:
https://ibb.co/TBT42T1
https://ibb.co/THD3Hsr

Reply:
OK. I thought it would save space, but it turned out that it's always just one row with attachments, and this makes it more readable. But please remove the right margin as well then